assets/docs/cosmo-group-ethz/models/kids-cgan/1.md

```diff
@@ -0,0 +1,41 @@
+# Module cosmo-group-ethz/models/kids-cgan/1
+
+A conditional GAN to generate KiDS-1000 weak lensing surveys.
+
+<!-- asset-path: https://people.phys.ethz.ch/~ipa/jafluri/cGAN/model/saved_model.tar.gz -->
+<!-- task: image-generator -->
+<!-- fine-tunable: false -->
+<!-- format: saved_model_2 -->
+<!-- https://colab.research.google.com/drive/1YF9OYu2EwAMLfW47jQdbthpADSfkPZPy?usp=sharing -->
+
+## Overview
+
+This is a conditional GAN to generate tomographic [KiDS](https://kids.strw.leidenuniv.nl/) weak lensing surveys using the same data and redshift distributions as described in [Asgari et al.](https://www.aanda.org/articles/aa/abs/2021/01/aa39070-20/aa39070-20.html) The cGAN generates the spherical data using [DeepSphere](https://github.com/deepsphere/deepsphere-cosmo-tf2) that treats the pixels on the sphere as nodes in a graph. The model and its training are described in detail in Yiu et al. (in prep.).
+
+### Example Use
+
+Please refer to our [colab notebook](https://colab.research.google.com/drive/1YF9OYu2EwAMLfW47jQdbthpADSfkPZPy?usp=sharing) for a detailed tutorial going through all functions of the model. The model generates mock KiDS-1000 surveys conditioned on the total matter density &Omega;<sub>M</sub> and fluctuation amplitude &sigma;<sub>8</sub>. Per default, the model takes a 2D array with shape (N,2) containing N sets of cosmoligical parameters (&Omega;<sub>M</sub>, &sigma;<sub>8</sub>) as an input.
+
+```
+# Load KiDS-cGAN model.
+model = hub.Module('https://tfhub.dev/cosmo-group-ethz/models/kids-cgan/1')
```

Comment:
Could you change this to `hub.load`, please? Your Colab also uses that and loading the model via `hub.Module` doesn't work since it's a TF2 SavedModel.